### PR TITLE
[DO-2133] [Contains Migration] Added Monetary asset model

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

**Review by roman-tiukh — APPROVED** [review]

File: business_register/models/declaration_models.py [hunk 1]
```diff
@@ -1323,6 +1323,58 @@ class Property(DataOceanModel):
     )
 
 
+class MonetaryAsset(DataOceanModel):
```
Comment (IegorSoboliev) [inline business_register/models/declaration_models.py:1326]: MonetaryAssets are our Money?

File: business_register/models/declaration_models.py [hunk 2]
```diff
@@ -1323,6 +1323,58 @@ class Property(DataOceanModel):
...
+        related_name='monetary_assets',
+    )
+    valuation = models.DecimalField(
+        max_digits=12,
+        decimal_places=2,
+        blank=True,
+        null=True,
+        help_text='valuation of the asset'
+    )
+    quantity = models.FloatField(
```
Comment (IegorSoboliev) [inline business_register/models/declaration_models.py:1357]: Can we also define a currency of cryptocurrency (Bitcoin, etc)

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/models/declaration_models.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py:1357]: this information is specified in 'descriptionObject' (description)

File: business_register/models/declaration_models.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py:1326]: I think, the name 'IntangibleAssets' would be better, but Anastasia said in the rules are the name monetary assets. Rename to IntangibleAssets?

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -193,7 +194,7 @@ def find_third_person_rights(self, right_data, object_name):
     def save_right(self, property, corporate_rights_data):
         if (
                 not corporate_rights_data.get('rights')
-                or corporate_rights_data.get('person') in self.NO_DATA
+                and corporate_rights_data.get('person') in self.NO_DATA
```
[the remark this answers is not in the document]
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py:197]: I think this is another topic for discussion)

**Review by Andrii17s — APPROVED** [review]

**Review by criticus — APPROVED** [review]

**Pull request merged**